doc: make sure the doc clearly states which ID should be passed on importing. (#86)
supabase/terraform-provider-supabase: docs/tutorial.md

Run `terraform -chdir=module apply`. Enter the ID of your Supabase project at the prompt. If your local TF state is empty, your project will be imported from remote rather than recreated.

> Alternatively, you may use the `terraform import ...` command without editing the resource file, for example, `terraform import supabase_project.production abcdefghijklmnopqrst` will let Terraform import your existing project, where `abcdefghijklmnopqrst` is your project's `Reference ID`. After which, running `terraform plan -out plan.out` may help you verify that your infrastructure matches the configuration.

Reviewer: ```suggestion
Alternatively, you may use the `terraform import ...` command without editing the resource file. For example, `terraform -chdir=module import supabase_project.production abcdefghijklmnopqrst` will let Terraform import your existing project with reference ID `abcdefghijklmnopqrst`. After import, you can run `terraform -chdir=module plan` to verify that your infrastructure matches the configuration.
```

Thank you for the suggestion on updating docs. I will let @charislam have another review on the copy.
